src/Uploadcare/Api.php

```diff
@@ -194,6 +197,12 @@ public function __construct(
         $this->cdn_secret_token = $cdn_secret_token;
```

Comment:
look at my previous comment
$cdn_secret_token should be null in constructor and we change default one only when it's passed as arg